moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp

```diff
@@ -236,6 +245,9 @@ class MoveItControllerManager : public moveit_controller_manager::MoveItControll
         getAbsName("controller_manager/list_controllers"));
     switch_controller_service_ = node_->create_client<controller_manager_msgs::srv::SwitchController>(
         getAbsName("controller_manager/switch_controller"));
+
+    std::scoped_lock<std::mutex> lock(controllers_mutex_);
```

Comment:
At L103, can you add a comment on what `controllers_mutex_` locks? I guess it is any of the member variables in this class.

Reply:
I added a comment -- you're right that it's just protecting access to member variables.